src/components/DatePicker/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { PropsWithChildren } from 'react';
+import { DatePickerProps } from './DatePickerProps';
+import { cx } from '@emotion/css';
+import { styDatePicker, styUndefinedDate } from './style';
+
+const DatePicker = (props: PropsWithChildren<DatePickerProps>): JSX.Element => {
+    const { placeholder, dateSize } = props;
+    const className =
+        dateSize !== undefined
+            ? `${dateSize} ${styDatePicker}`
+            : cx(styDatePicker, styUndefinedDate);
+
+    return (
+        <div>
+            <input
+                className={className}
+                placeholder={placeholder}
+                type="date"
+                name=""
+                id=""
```

Comment:
why this has `magic string`

you can add comment to mark it as draft or WIP (work in progress) 
for example

`// TODO: this is still WIP, will populate with real data later on`

Reply:
this will make sure you won't forget about those things and miss them out.